lib/ObjWriter/objwriter.cpp

```diff
@@ -702,9 +702,33 @@ void ObjectWriter::EmitDebugLoc(int NativeOffset, int FileId, int LineNumber,
   }
 }
 
+void ObjectWriter::EmitCVUserDefinedTypesSymbols() {
```

Comment:
Out of interest, what do we do if there are no UDTs? Should we skip this altogether and save the bytes?